Tidy up readme.md

hamishwillee opened the pull request:
This updates the project readme.md to good standard for release. It uses a similar structure to the readme's for dk-android and dk-python.

It is currently in draft, and will need updates after the following are known:
* [x] Is there a Gitter channel - what is URL?
* [x] Is there a support forum - what is URL?
* [x] There is a CI test - is that just linux - use the linux icon?
* [x] Are there windows/mac tests. Can we have shield/link for them?

A lot of information on how to create analysers and missing analyzers has been removed - there are now links to the official documentation on this.

mrpollo commented:
Gitter: https://gitter.im/dronekit/dronekit-la

mrpollo commented on README.md:
```diff
@@ -1,79 +1,82 @@
-# DroneKit LA
+# DroneKit LogAnalyzer (LA)
```
I've been writing this the opposite way and with a space between Log Analyzer **DroneKit LA (Log Analyzer)**, I think it reads better and it differentiates itself further from the current **LogAnalyzer** in Ardupilot

hamishwillee replied:
OK, I'll update that.

mrpollo commented on README.md:
```diff
@@ -1,79 +1,82 @@
-// - two fundamental types of test
-//  - is the software working correctly (EKF issues)
-//  - is the vehicle doing sensible things (attitude etc)
+Copyright 2016 3D Robotics, Inc.
```
Actually I'm not so sure about this, can we leave it out by now?

hamishwillee replied:
@mrpollo - you're suggesting we lose the copyright notice?

hamishwillee commented:
Updated with gitter/support icons/links. Fixed title. Need clarity on:

* whether you "really" want to remove copyright notice
* what is the setup on mac to build dk-la.

This would be OK to go out live right now though.

mrpollo commented:
Mac tests https://travis-ci.org/dronekit/dronekit-la

mrpollo commented:
I updated the badges, they are all shields and they are all showing up good now, I'm going to proceed with merging
